src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/SearchOptionsFactoryTests.cs

```diff
@@ -44,6 +46,7 @@ public SearchOptionsFactoryTests()
             _factory = new SearchOptionsFactory(
                 _expressionParser,
                 () => searchParameterDefinitionManager,
+                new OptionsWrapper<CoreFeatureConfiguration>(new CoreFeatureConfiguration()),
```

Comment:
Should we add tests that validate that the `IncludeTotal` value is set as expected when there are no search query parameters?